common/djangoapps/student/models.py

```diff
@@ -3461,6 +3463,27 @@ def should_celebrate_first_section(enrollment):
         except CourseEnrollmentCelebration.DoesNotExist:
             return False
 
+    @staticmethod
+    def should_celebrate_weekly_goal(enrollment):
+        """ Returns the celebration value for weekly_goal with appropriate fallback if it doesn't exist """
+        if not enrollment:
+            return False
+        try:
+            if enrollment.celebration.celebrate_weekly_goal:
+                # Avoiding circular import
+                from lms.djangoapps.course_goals.models import CourseGoal, UserActivity
```

Comment:
This is just a convention nit, but I usually see this sort of import at the top of the method. Similar to top of module imports.